CHANGELOG.md

```diff
@@ -5,6 +5,11 @@
 - [Amazon Linux 2 release notes](https://docs.aws.amazon.com/AL2/latest/relnotes/relnotes-al2.html)
 - [Amazon Linux AMI 2018.03 Release Notes](https://aws.amazon.com/amazon-linux-ami/2018.03-release-notes/)
 
+## 20250318
+- ecs version: 1.91.1
+- bug fix: Remove AL1 AMI build automation [#406](https://github.com/aws/amazon-ecs-ami/pull/406)
```

Comment:
this is not a bug-fix right? can keep it in the changelog as "housekeeping" or remove it completely

Reply:
Discussed offline - based off [documentation](https://aws.amazon.com/blogs/containers/amazon-ecs-optimized-amazon-linux-ami-end-of-life/), removing AL1 AMI build automation seems to have been an outstanding miss from our side. We should have likely stopped including AL1 in release scope a long time ago along with when the maintenance support period for the AL1 ECS-optimized AMI ended on June 30, 2023.